src/Apps/Search/Components/GenericSearchResultItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { Flex, Image, Link, Sans, Serif, Spacer } from "@artsy/palette"
+import React, { SFC } from "react"
+
+interface GenericSearchResultItemProps {
+  item: any
+  index: number
+}
+
+export const GenericSearchResultItem: SFC<
+  GenericSearchResultItemProps
+> = props => {
+  const { item, index } = props
+
+  return (
+    <>
+      <Flex flexDirection="row">
+        <div>
```

Comment:
Rather than `<div>`, we should try to use things from our design system. So `<Box>` maybe? (or `<>` if it doesn't even need its own container).